### ISSUE TITLE
Import bank statements and other transactions

### ISSUE THREAD

**Issue by jessicana** [prashants/webzash#163]
Are there any plans for adding import bank statements and transactions from excel? For example, if I have monthly bank statements, can I import them in excel instead of recording transactions manually?

**Comment by rakeshpatil1983** [comment]
Yes, I like this to be added too. I'm willing to help contribute to this. This will reduce the time to add manual entries.

**Comment by prashants** [comment]
If you can provide me with a sample bank statement I can think about how to
do it

**Comment by rakeshpatil1983** [comment]
A typical statement is as attached. 


[51068066_1667831974802.xls](https://github.com/prashants/webzash/files/9952660/51068066_1667831974802.xls)
